README.md

```diff
@@ -494,6 +494,25 @@ If you don't have a bridging header, you can go to your target's `Build Settings
 
 ** Comparison with RAC with respect to v3.0-RC.1
 
+### Is RxSwift Functional Reactive Programming?
+
+Why this question is important? Because RxSwift is **NOT** *Functional Reactive Programming*. This clarification is important for 2 big reasons:
+
+* the Rx world is not time-continuous and doesn't have a denotative semantics, so all fundamentals of FRP are missing
+* concepts and names are different, so when you start to learn Rx you should not link Rx and FRP
+
+If you are interested in learning more about what FRP is and to understand why is different from the Reactive Extensions this list of resources might help:
+
+* [Functional Reactive Animations](http://conal.net/papers/icfp97/)
+* [Essence and Origins of FRP](http://conal.net/talks/essence-and-origins-of-frp-lambdajam-2015.pdf)
```

Comment:
I suggest https://github.com/conal/talk-2015-essence-and-origins-of-frp instead, to include abstract and video(s), as well as the slides PDF. The Lambda Jam 2015 video will be linked there as well when ready.